applications/altran-v2x/readme.md

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2020 Altran Technology
+
+Overview of Altran V2X Apps:
+=============================
+
+Altran V2X solution enables safety & non-safety applications for V2V,V2I,V2P and V2N scenarios for Road Side Units (RSU), On Board Units (OBU) and Smart Pedestrian Devices . The solution comes with application and underlying stack conformant to regional standards - WAVE / ETSI and China.
+
+V2X Application Description:    
+==============================
+Following Vehicle To Infrastructure (V2I) applications are supported:
+
+Roadside Alert (RSA): RSU periodically broadcast advisory messages about the road hazards (Slippery Road, Animal On the Road etc.)
```

Comment:
Would you like to make as bullet points?
Should add `* ` in front of each element in the list.